roles/libvirt-setup/defaults/main.yml

```diff
@@ -15,6 +15,9 @@ libvirt_nic_model: virtio
 # Set to false for DHCP-less testing with static-ips
 libvirt_network_dhcp: true
 
+# Set to true for dual-stack testing with ipv6
+libvirt_network_ipv6: false
```

Comment:
I'd feel courageous here and enable it by default probably. And maybe add two flags to disable IPv4 and IPv6 when needed, so that we can easily enable single stack testing for both. What do you think?

Reply:
I would prefer to leave it v4 only by default, just so for debugging the environment is as simple as possible